helm/dbp-keycloak/templates/ingress2nd.yaml

```diff
@@ -2,28 +2,31 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ template "common.names.name" . }}-2nd
+  name: {{ template "common.names.fullname" . }}-2nd
   namespace: {{ template "common.names.namespace" . }}
-  labels: 
+  labels:
     {{- include "common.labels" . | nindent 4 }}
   {{- with .Values.ingress.annotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
 spec:
+  {{- if .Values.ingress.tls.enabled }}
   tls:
     - hosts:
-        - {{ .Values.keycloak2ndHostname  }}
+        - {{ .Values.keycloak.secondaryHostname  }}
+      secretName: "{{ .Values.ingress.tls.secretName }}"
```

Comment:
the setting for secretName shoud be optional, if you use an wildcard cert who is not store in the same namespace as keycloak it's complicated